x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -151,31 +146,24 @@ string
     : STRING
     ;
 
-AND: 'AND';
-ANY: 'ANY';
-ASC: 'ASC';
-BETWEEN: 'BETWEEN';
-BY: 'BY';
-CHILD: 'CHILD';
-DESCENDANT: 'DESCENDANT';
-EVENT: 'EVENT';
-FALSE: 'FALSE';
-IN: 'IN';
-JOIN: 'JOIN';
-MAXSPAN: 'MAXSPAN';
-NOT: 'NOT';
-NULL: 'NULL';
-OF: 'OF';
-OR: 'OR';
-SEQUENCE: 'SEQUENCE';
-TRUE: 'TRUE';
-UNTIL: 'UNTIL';
-WHERE: 'WHERE';
-WITH: 'WITH';
+AND: 'and';
```

Comment:
The convention in ANTLR is to declare tokens by upper-case, just like their declaration. It also helps with readability.

Reply:
I think this a grammar specific thing, though. The token name AND is capital but the string it matches is lowercase, because EQL keywords are all lowercase and case sensitive. Similar to python's grammar with lowercase `and`, `or`, `not`, etc.

https://github.com/antlr/grammars-v4/blob/master/python3/Python3.g4#L315